src/utils/options.js

```diff
@@ -11,6 +11,15 @@ const passphraseDescription = `Specifies a source for your secret passphrase. Li
 	- --passphrase stdin (takes the first line only)
 `;
 
+const secondPassphraseDescription = `Specifies a source for your second secret passphrase. Lisky will prompt you for input if this option is not set. Source must be one of \`env\`, \`file\` or \`stdin\`. Except for \`stdin\`, a corresponding identifier must also be provided.
+
+	Examples:
+	- --second-passphrase 'pass:my second secret passphrase' (should only be used where security is not important)
+	- --second-passphrase env:SECOND_SECRET_PASSPHRASE
+	- --second-passphrase file:/path/to/my/secondPassphrase.txt (takes the first line only)
+	- --second-passphrase stdin (takes the first line only)
```

Comment:
`takes one line only` - this might have to change in the `passphraseDescription` too.